components/StarSearch/ShareChatMenu.tsx

```diff
@@ -35,8 +35,8 @@ export function ShareChatMenu({ shareUrl, copyLinkHandler, createLink }: ShareMe
   }
 
   return (
-    <DropdownMenu open={dropdownOpen} modal={false}>
-      <DropdownMenuTrigger onClick={() => setDropdownOpen(!dropdownOpen)} aria-label="Share prompt options">
+    <DropdownMenu open={dropdownOpen} modal={false} onOpenChange={(open) => setDropdownOpen(open)}>
```

Comment:
nit: this works the same as the next code: 
```suggestion
    <DropdownMenu open={dropdownOpen} modal={false} onOpenChange={setDropdownOpen}>
```

Reply:
@a0m0rajab done.